Makefile

```diff
@@ -53,3 +53,7 @@ checksum:
 	sha256sum target/release/basilisk > target/release/basilisk.sha256
 	cp target/release/wbuild/basilisk-runtime/basilisk_runtime.compact.compressed.wasm target/release/
 	sha256sum target/release/basilisk_runtime.compact.compressed.wasm > target/release/basilisk_runtime.compact.compressed.wasm.sha256
+
+release: build checksum
+
+all: test test-benchmarks build build-benchmarks checksum
```

Comment:
IMHO `test` target can be removed because `test-benchmarks` runs all the tests + benchmarking tests. 
Also, what is this target for? Maybe also include `format` and `clippy`.

Reply:
csure, i forgot clippy, i use it as my little ci :)